PROCESS.md

```diff
@@ -0,0 +1,49 @@
+# Process
+
+## Preface
+
+This document is set out to describe the processes and guidelines for cloud based resource management related to CNCF credits and the CNCF credits program.
+
+A great deal is not automated or not yet possible to automate.
+
+repos:
+
+- [cncf/credits](https://github.com/cncf/credits) :: submissions, tickets and Terraform
+- [cncf-infra/aws-infra](https://github.com/cncf-infra/aws-infra) :: Terraform for AWS
```

Comment:
Bit of a tangent, is `cncf-infra` part of the CNCF enterprise account? If not, we should move it and ensure it follows all our standards.

Reply:
cncf-infra is not currently, it's been a bit of a development area. We can clean it up and add it. Do we have a documented set of standards we follow as CNCF?